spec/Section 5 -- Validation.md

```diff
@@ -677,14 +715,42 @@ fragment argOnOptional on Dog {
 }
 ```
 
-the following is invalid since `command` is not defined on `DogCommand`.
+The above is also applicable to fragment-definitions and fragment-spreads, each
+variable must be defined by the fragment-definition before it can be inserted as
+an argument by the fragment-spread.
```

Comment:
```suggestion
The above is also applicable to fragment definitions and fragment spreads, each
variable must be defined by the fragment definition before it can be inserted as
an argument by the fragment spread.
```